File: practices/time_of_day.py
# MA 6th time of day

import datetime

current = datetime.datetime.now()
hour = current.hour
print(f"The time is: {current}")
print(f"hour is: {hour}")


if hour >= 1 and hour <= 11:
    print("Good Morning!")
elif hour >= 12 and hour <= 16:
    print("Good Afternoon!")
else:
    if hour >= 17 and hour <= 24:
        print("Good Evening!")

File: c_programming/projects/timeofday.c
// MA 6th Time of day project for c
#include <stdio.h>
#include <string.h>

int main (void){

    int hour;
    printf("What hour is it in millitary time: ");
    scanf("%d", &hour);

    if(hour >= 1 && hour <= 11){
        printf("Good morning!\n");
    }else if(hour >= 12 && hour <= 16){
        printf("Good afternoon!\n");
    }else{
        printf("Good evening!\n");
    }

    return 0;
}